lib/auth/httpfallback.go

```diff
@@ -453,107 +451,3 @@ func (c *Client) DeleteTrustedCluster(ctx context.Context, name string) error {
 	_, err := c.Delete(ctx, c.Endpoint("trustedclusters", name))
 	return trace.Wrap(err)
 }
-
-// DeleteAllNodes deletes all nodes in a given namespace
-func (c *Client) DeleteAllNodes(ctx context.Context, namespace string) error {
-	if err := c.APIClient.DeleteAllNodes(ctx, namespace); err != nil {
-		if !trace.IsNotImplemented(err) {
-			return trace.Wrap(err)
-		}
-	} else {
-		return nil
-	}
-
-	_, err := c.Delete(ctx, c.Endpoint("namespaces", namespace, "nodes"))
-	if err != nil {
-		return trace.Wrap(err)
-	}
-	return nil
-}
-
-// DeleteNode deletes node in the namespace by name
-func (c *Client) DeleteNode(ctx context.Context, namespace string, name string) error {
-	if err := c.APIClient.DeleteNode(ctx, namespace, name); err != nil {
-		if !trace.IsNotImplemented(err) {
-			return trace.Wrap(err)
-		}
-	} else {
-		return nil
-	}
-
-	_, err := c.Delete(ctx, c.Endpoint("namespaces", namespace, "nodes", name))
-	if err != nil {
-		return trace.Wrap(err)
-	}
-	return nil
-}
-
-type nodeClient interface {
-	ListNodes(ctx context.Context, req proto.ListNodesRequest) (nodes []types.Server, nextKey string, err error)
-	GetNodes(ctx context.Context, namespace string, opts ...services.MarshalOption) ([]types.Server, error)
-}
-
-// GetNodesWithLabels is a helper for getting a list of nodes with optional label-based filtering.  This is essentially
-// a wrapper around client.GetNodesWithLabels that performs fallback on NotImplemented errors.
-//
-// DELETE IN 11.0.0, this function is only called by lib/client/client.go (*ProxyClient).FindServersByLabels
```

Comment:
Is this safe to delete now even though it's marked for 11?

Reply:
Yes, `GetNodesWithLabels` and `GetNodes` were both marked as 11.0.0 after the fact (since Forrest and I didn't mark them at the time), but they were actually added in `v8` and `v6` respectively and can safely be deleted.